setup.py

```diff
@@ -238,7 +238,7 @@ def isgood(name):
     'package_data'      :  {'': ['*.sh', '*.json', 'VERSION', 'SDIST']},
 
     #'install_requires'  :  ['radical.pilot', 'pika', 'pandas', 'numpy', 'matplotlib'],
-    'install_requires'  :  ['radical.utils', 'pika', 'radical.pilot',
+    'install_requires'  :  ['radical.utils', 'pika==0.13.1', 'radical.pilot',
```

Comment:
Why is this dependency introduced again?

Reply:
Oh oh, which version exists on conda-forge? The latest pika will fail due to change in the argument order: https://github.com/radical-cybertools/radical.entk/issues/331

Reply:
```bash
(base) iparaskmac:two.radical iparask$ conda search pika -c conda-forge
Loading channels: done
# Name                       Version           Build  Channel
pika                          0.10.0          py27_0  conda-forge
pika                          0.10.0          py34_0  conda-forge
pika                          0.10.0          py35_0  conda-forge
pika                          0.10.0          py36_0  conda-forge
pika                          0.11.0          py27_0  conda-forge
pika                          0.11.0          py35_0  conda-forge
pika                          0.11.0          py36_0  conda-forge
pika                          0.11.2          py27_0  conda-forge
pika                          0.11.2          py35_0  conda-forge
pika                          0.11.2          py36_0  conda-forge
pika                          0.12.0          py27_0  conda-forge
pika                          0.12.0          py35_0  conda-forge
pika                          0.12.0          py36_0  conda-forge
pika                          0.12.0            py_1  conda-forge
pika                          0.13.0            py_0  conda-forge
pika                         1.0.0b1            py_0  conda-forge
pika                         1.0.0b2            py_0  conda-forge
pika                           1.0.0            py_0  conda-forge
pika                           1.0.1            py_0  conda-forge
```